ward/tests/test_fixtures.py

```diff
@@ -217,3 +223,35 @@ def fix():
         pass
 
     assert len(Fixture(fix).parents()) == 0
+
+
+@test("fixture_parents_and_children analyzes fixture dependencies correctly")
+def _():
+    @fixture
+    def a():
+        pass
+
+    @fixture
+    def b():
+        pass
+
+    @fixture
+    def c(a=a, b=b):
+        pass
+
+    @fixture
+    def d(a=a, c=c):
+        pass
+
+    fa, fb, fc, fd = fixtures = list(map(Fixture, (a, b, c, d)))
```

Comment:
Mostly a matter of preference, but I'd like it if we could stick to comprehensions instead of map:

```suggestion
    fa, fb, fc, fd = fixtures = [Fixture(f) for f in (a, b, c, d)]
```

Reply:
:+1: , I agree that it looks better that way. Sometimes I get the urge to be clever like that, and I'm almost always wrong :)